includes/polling/os/patton-sn.inc.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+//PATTON SmartNode
+$oids = array(
+    'hw_release' => '.1.3.6.1.4.1.1768.100.1.3.0',
```

Comment:
You are going to hate me... But can you move this array() to []?

Reply:
No problem, I sometimes see those, but usually not :) On its way !